convolutional_neural_network.cpp

```diff
@@ -444,35 +401,19 @@ class Relu : public Act {
 };
 
 class Flatten : public Layer {
-  // Flattens to a row vector
+  // Flattens to a column vector
```

Comment:
Adopting the following convention: a vector<double> is a column vector

So we multiply it on the right with the weights matrix in the Dense's h, i.e., W*v